database/BUILD

```diff
@@ -54,17 +54,17 @@ native_java_libraries(
     mac_deps = [
         "@maven//:com_google_ortools_ortools_darwin",
         "@maven//:com_google_ortools_ortools_darwin_java",
-        "@maven//:org_rocksdb_rocksdbjni",
```

Comment:
how is it possible that before we could depend on an un-versioned rocksdbjni and now we need to use a versioned speedbjni?